includes/bootstrap.inc

```diff
@@ -26,6 +26,12 @@ define('DRUSH_BOOTSTRAP_MAX', -2);
  */
 define('DRUSH_BOOTSTRAP_DRUSH', 0);
```

Comment:
In a followup, I'd like to get rid of this constant. We don't "bootstrap" Drush anymore. We preflight Drush, and we bootstrap Drupal. Lets work on making that distinction complete.